### ISSUE TITLE
Keyboard navigation, focus order and visible focus

### ISSUE THREAD

**Issue by markusjt** [cessda/cessda.cdc.versions#562]
[2.1.1: Keyboard](https://www.w3.org/WAI/WCAG21/Understanding/keyboard.html): Some elements (like filters at least) are being skipped by keyboard navigation.
[2.4.3: Focus Order](https://www.w3.org/WAI/WCAG21/Understanding/focus-order.html): The order is currently okay but some elements are skipped that shouldn't be so need to make sure it's still good after more elements can be navigated with keyboard.
[2.4.7: Focus Visible](https://www.w3.org/WAI/WCAG21/Understanding/focus-visible.html): Some elements are completely missing any indication that they have focus even though they can be navigated to with keyboard (which is good so need to fix focus to be visible).
